mst/dblink-extension.md

```diff
@@ -53,13 +53,13 @@ To enable the `dblink` extension on an MST PostgreSQL service:
 1.  Create a table named `inventory`:
 
 ```sql
```

Comment:
Hey Rajie, sorry should have been more clear, the backtick lines need to be aligned with the list indentation as well. Otherwise LGTM

Reply:
Thank you @charislam !!! fixed it now.